### ISSUE TITLE
Unable to export Spotify Playlist to YouTube Music

### ISSUE THREAD

**Issue by Akalsharan** [sigma67/spotifyplaylist_to_ytmusic#43]
Hello

I've tried a few different playlist URLs and keep getting the following error. Is someone able to advise on what's wrong, please? Using Termux on Android 11

Traceback (most recent call last):
  File "/data/data/com.termux/files/home/spotifyplaylist_to_ytmusic/YouTube.py", line 190, in <module>
    main()
  File "/data/data/com.termux/files/home/spotifyplaylist_to_ytmusic/YouTube.py", line 183, in main
    ytmusic.add_playlist_items(playlistId, videoIds)
  File "/data/data/com.termux/files/home/spotifyplaylist_to_ytmusic/YouTube.py", line 91, in add_playlist_items
    self.api.add_playlist_items(playlistId, videoIds)
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/ytmusicapi/mixins/playlists.py", line 297, in add_playlist_items
    raise Exception(
Exception: You must provide either videoIds or a source_playlist to add to the playlist

**Comment by sigma67** [comment]
Please provide the command you ran and the source playlist

**Comment by Akalsharan** [comment]
> Please provide the command you ran and the source playlist

Thanks for responding so quickly.

I've tried a few different playlists. It outputs with 10 searched, 20 searched etc, until the error. I've even tried some with only 1/2 tracks,

This is the most recent one I've tried:

python YouTube.py https://open.spotify.com/playlist/3M73k43vXWTrE0QPHGhTIV

I've also tried using python3 instead of python.

Any thoughts?
